tools/scripts/updateAnimationAliases.rb

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env ruby
+require 'json'
+require 'optparse'
+require_relative '../../lib/cdo/cdo_cli'
+include CdoCli
+
+#
+# This script is for updating the aliases in the metadata for each animation
+# using spriteCostumeLibrary as the source of truth.
+# Usage:
+# 1. Download the Sprite Lab animation library from S3 and put it in this directory
+# 2. Make any desired changes to the aliases in cdo/apps/srce/p5lab/spritelab/spriteCostumeLibrary
+# 3. Run ./updateAnimationAliases
+# 4. Upload the new animation metadata files to S3
+#
+
+SPRITELAB_MANIFEST_FILE = '../../apps/src/p5lab/spritelab/spriteCostumeLibrary.json'.freeze
+SPRITELAB_ANIMATION_LIBRARY_PATH = './cdo-animation-library/spritelab/'.freeze
+
+GAMELAB_MANIFEST_FILE = '../../apps/src/p5lab/gamelab/animationLibrary.json'.freeze
+GAMELAB_ANIMATION_LIBRARY_PATH = './cdo-animation-library/gamelab/'.freeze
+
+class AliasUpdater
+  def initialize(options)
+    @options = options
+  end
+
+  def get_update_aliases_for_animation(all_aliases, animation_name)
+    aliases = []
+    all_aliases.each do |aliaz, animations|
```

Comment:
NIT: Should `aliaz` be `alias`?

Reply:
`alias` is a keyword in Ruby so I copied this workaround from rebuildAnimationLibraryManifest.rb